### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by zhirui-gao** [jinlinyi/SparsePlanes#7]
#### Hi,thanks for your great work!
I home some trouble for training the network on step1(the detect plane stage) because I am not sure when to stop! I am training the network on Scannet dataset using your pre-trained model,and my loss is continuing to decline slowly during 40k iteration. 
 Could you give me some advice?Thank you!
![2022-03-12 16-27-06 的屏幕截图](https://user-images.githubusercontent.com/50123492/158010482-24d7e733-0e35-4001-b85f-1624856c2a8d.png)

**Comment by zhirui-gao** [comment]
And another one to ask for advice。
The learning rate is 1e-3 now.And for fine-tune task,Whether it is relatively large?

**Comment by jinlinyi** [comment]
Hi, thank you for your interest in our project! 

We didn't train on ScanNet in our project. Do you have a validation set during training? You need to look at the loss and accuracy when you evaluate your model on the validation set to determine the time to stop. The loss on the training set is expected to go down and may lead to overfitting. 
For the learning rate, unfortunately, I cannot give you much advice since the learning rate depends on lots of things -- the batch size, the optimizer, the dataset size, etc. 

Best,
Linyi

**Comment by zhirui-gao** [comment]
Thanks a lot!
